src/blockencodings.cpp

```diff
@@ -107,12 +107,13 @@ ReadStatus PartiallyDownloadedBlock::InitData(const CBlockHeaderAndShortTxIDs& c
     std::vector<bool> have_txn(txn_available.size());
     {
     LOCK(pool->cs);
-    for (size_t i = 0; i < pool->vTxHashes.size(); i++) {
-        uint64_t shortid = cmpctblock.GetShortID(pool->vTxHashes[i].first);
+    for (size_t i = 0; i < pool->txns_randomized.size(); i++) {
+        const auto& tx = pool->txns_randomized[i];
```

Comment:
nit (although I realize this diff is from a scripted-diff, so may not be worth updating here)
```suggestion
    for (const auto& tx : pool->txns_randomized) {
```